frontend/packages/topology/src/components/list-view/TopologyListViewNode.tsx

```diff
@@ -134,8 +134,8 @@ const TopologyListViewNode: React.FC<TopologyListViewNodeProps & DispatchProps>
   return (
     <DataListItem
       key={item.getId()}
-      id={item.getId()}
-      aria-labelledby={`${item.getId()}_label`}
+      id={`${item.getId()}`}
+      aria-label={`${item.getId()}_label`}
```

Comment:
Similar issue here. "1234_label" is not a useful label. We should keep the dom reference here. Please checkout the accessibility props in Firefox:

![image](https://user-images.githubusercontent.com/139310/162679969-5d490e3f-f6d4-4910-9c82-6248b9bccd28.png)

Before:

![image](https://user-images.githubusercontent.com/139310/162680035-70c5999f-7134-414c-8e21-62a70b0a1d33.png)

After:

![image](https://user-images.githubusercontent.com/139310/162680103-34bbbf08-d29f-4420-acd4-4d0c76516c19.png)